homeassistant/components/modbus/modbus.py

```diff
@@ -150,10 +151,6 @@ def __init__(self, client_config):
         else:
             # network configuration
             self._config_host = client_config[CONF_HOST]
-            self._config_delay = client_config[CONF_DELAY]
-
-        if self._config_delay > 0:
-            _LOGGER.warning("Parameter delay is accepted but not used in this version")
```

Comment:
Was this an old feature that we had deprecated?

Reply:
yes a leftover from when we tried async pymodbus.